editor/src/components/canvas/canvas.ts

```diff
@@ -131,16 +131,10 @@ const Canvas = {
     }
 
     return Utils.flatMapArray((rootComponent) => {
-      if (rootComponent.globalFrame == null) {
-        return []
-      } else {
-        const nonNullGlobalFrame = rootComponent.globalFrame
-        const rootElements = MetadataUtils.getImmediateChildren(metadata, rootComponent.scenePath)
-        return flatMapArray(
-          (root) => recurseChildren(nonNullGlobalFrame, root).frames,
-          rootElements,
-        )
-      }
+      const nonNullGlobalOffset: CanvasVector =
+        rootComponent.globalFrame ?? ({ x: 0, y: 0 } as CanvasVector)
```

Comment:
I spent a bit of time trying to fix the actual `globalFrame` of the Storyboard `rootComponent`, but the potential fix involves touching quite a bit of the scene/storyboard hacky code in `useGetStoryboardRoot` https://github.com/concrete-utopia/utopia/blob/4d7507afd73dfb0c7175befa7d286318b284327a/editor/src/components/canvas/ui-jsx-canvas.tsx#L416-L424

so instead I just hacked a fallback in here and added a TODO there.

I think it's not worth touching as the potential risk for regressions is too large, and I don't mind this 0;0 vector as a fallback value here